### ISSUE TITLE
Don't Understand How to Install

### ISSUE THREAD

**Issue by Highlander-bdj** [Floppy/van_dam#1163]
I'd love to give this software a try, but I do not understand the installation instructions.  If there were more detailed instructions geared to someone who doesn't know how to use docker, that would be very useful in broadening your audience.

**Comment by Highlander-bdj** [comment]
Thank you for considering.  I don't know if it's even possible, as I don't know docker at all but maybe a preconfigured docker container that could just be downloaded and run via docker desktop, etc.?

**Comment by Floppy** [comment]
yeah, it should be that simple, but needing separate databases etc is an issue (see #27, I've wanted to solve that for a long time). I'll see what I can do.
